meson.build

```
@@ -432,10 +433,29 @@ if build_tools
         dependencies: dep_libxkbcommon,
     )
 
+    # Tool: xkbcli
     executable('xkbcli', 'tools/xkbcli.c',
                dependencies: tools_dep, install: true)
     install_man('tools/xkbcli.1')
 
+    bash_completion_path = get_option('bash-completion-path')
+    if bash_completion_path == ''
+        bash_completion = dependency('bash-completion', required : false)
```

```suggestion
        bash_completion = dependency('bash-completion', required: false)
```
mostly basing this on the context of this diff, maybe we have spaces before `:` elsewhere? if so we should probably normalize this

---

Fixed